templates/registration/login.html

```diff
@@ -16,25 +16,34 @@
   {% endif %}
 {% endif %}
 
-<form method="post" action="{% url 'login' %}">
+<form method="post" action="{% url 'login' %}" class="uk-form uk-form-horizontal maxw600">
 {% csrf_token %}
 
-<div>
-  <td>Email:</td>
-  <td>{{ form.username }}</td>
-</div>
-<div>
-  <td>{{ form.password.label_tag }}</td>
-  <td>{{ form.password }}</td>
-</div>
-
-<div>
-  <input type="submit" value="login" />
-  <input type="hidden" name="next" value="{{ next }}" />
-</div>
+      <div class="uk-form-row uk-margin">
+          <label class="uk-form-label" for="id_username">
+          Email: </label>
+          <div class="uk-form-controls">
+            <input class="uk-input" type="email" name="username" autofocus required id="id_username">
+            {{ field.errors }}
```

Comment:
does this need to be `form.email.errors`?

Reply:
Oh thanks for spotting this! - I think it needs to be form.username.errors actually because of the USERNAME_FIELD = email. Also this form never seems to raise any other error but "Your username and password didn't match. Please try again." .. and that's not even a form error I think :) But I have changed it now, just in case